src/util/Util.js

```diff
@@ -174,6 +174,51 @@ class Util {
         return webpMedia;
     }
 
+    /**
+     * Formats a media to ogg/opus
+     * @param {MessageMedia} media
+     * 
+     * @returns {Promise<MessageMedia> media} in opus/ogg (audio) format
+     */
+    static async formatToOpusAudio(media) {
+        if (!media.mimetype.includes('audio'))
+            throw new Error('media is not an audio');
+
+        if (media.mimetype === 'audio/ogg; codecs=opus')
+            return media
+
+        const tempFile = path.join(
+            tmpdir(), `${Crypto.randomBytes(6).readUIntLE(0, 6).toString(36)}.ogg`
+        );
+
+        const stream = new (require('stream').Readable)();
+        const buffer = Buffer.from(
+            media.data.replace(`data:${media.mimetype};base64,`, ''),
+            'base64'
+        );
+        stream.push(buffer);
+        stream.push(null);
+
+        await new Promise((resolve, reject) => {
+            ffmpeg(stream)
+                .on('error', reject)
+                .on('end', () => resolve(true))
```

Comment:
Maybe on error try send the bad format?

Reply:
Not possible. If you try to send the wrong format, while trying to send audio/ptt, it'll crash. The better solution is to throw an error and let this decision be made in a try-catch block, out of this routine.